src/libs/antares/args/include/antares/args/args_to_utf8.h

```diff
@@ -1,11 +1,14 @@
 #pragma once
 
+#include <utility>
+
 class IntoUTF8ArgsTranslator
 {
 public:
-	IntoUTF8ArgsTranslator(int argc, char**& argv);
+	IntoUTF8ArgsTranslator(int argc, char** argv);
```

Comment:
It seems we have an indentation inconsistency here

Reply:
done (applied clang-format)